apps/back/src/Command/ViewBalanceCommand.php

```diff
@@ -47,27 +47,32 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         $benoit->addExpense($expense4);
 
         //Balance
-        foreach($users as $user){
+        foreach($users as $user)
+        {
             $expensesPaidByUser = [];
-            foreach($expenses as $expense){
-                if($expense->getPayer() == $user){
+            foreach($expenses as $expense)
+            {
+                if($expense->getPayer() == $user)
+                {
                     $expensesPaidByUser [] = $expense->getAmount();
                 }
             } 
             $sumExpensesPaidByUser = array_sum($expensesPaidByUser);
 
             $expensesIncludingUser = [];
-            foreach($expenses as $expense){
-                if(in_array($user, ($expense->getParticipants()))){
+            foreach($expenses as $expense)
+            {
+                if(in_array($user, ($expense->getParticipants())))
+                {
                     $expensesIncludingUser [] = $expense->getAmount()/(count($expense->getParticipants()));
                 }
             }
             $sumExpensesIncludingUser = array_sum($expensesIncludingUser);
 
             $balancePerUser = $sumExpensesPaidByUser - $sumExpensesIncludingUser;
-            dump($user->getName().' '.$balancePerUser);
+            $output->writeln("{$user->getName()}"." "."{$balancePerUser}");
```

Comment:
```suggestion
            $output->writeln("{$user->getName()} {$balancePerUser}");
```
Entre double-quote " tu peux insérer des littéraux PHP sans avoir à concaténer des chaînes pour mettre des espaces, ça ajoute un peu de lisibilité selon les préférences.